### PR TITLE
feat: add chart time period in the query string url. 

### PR DESCRIPTION
**Pull request by ceoaliongroo**
#404

### CODE REVIEW

**Comment by ceoaliongroo** [comment]
Locally pass the test:
screen shot 2015-03-22 at 5 32 55 pm

File: client/app/scripts/app.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ angular
         controller: 'DashboardCtrl'
       })
       .state('dashboard.withAccount', {
-        url: 'dashboard/{accountId:int}?:chartFreq',
+        url: 'dashboard/{accountId:int}?{chartFreq:int}&chartNextPeriod&chartPreviousPeriod',
```
Comment (amitaibu) [inline client/app/scripts/app.js]: ``:int`` all the arguments

File: client/app/scripts/controllers/usage.js [hunk 2]
```diff
@@ -22,39 +23,45 @@ angular.module('negawattClientApp')
...
-            $scope.isLoading = false;
-          });
-
-          $location.search('chartFreq', $stateParams.chartFreq);
-          $urlRouter.update(true);
+    $scope.changeFrequency = function() {
+      var params = {};
+
+      // Change of frequency.
+      if ( this.frequencies && ($stateParams.chartFreq !== this.frequencies[this.$index].type) ) {
```
Comment (amitaibu) [inline client/app/scripts/controllers/usage.js]: add comment

File: client/app/scripts/controllers/usage.js [hunk 3]
```diff
@@ -22,39 +23,45 @@ angular.module('negawattClientApp')
...
+        $stateParams.chartFreq = this.frequencies[this.$index].type;
+        // Delete properties only if not exist in the Url.
+        if (chartUpdated) {
+            delete $stateParams['chartNextPeriod'];
```
Comment (amitaibu) [inline client/app/scripts/controllers/usage.js]: indentation error

**Comment by ceoaliongroo** [comment]
Merged.

**Pull request closed without merging**